addon/reducers/frost-select.js

```diff
@@ -219,6 +219,9 @@ export default function reducer (state, action) {
       nextState = select(state, action.itemIndex)
       break
     case SELECT_HOVER:
+      if (state.hoveredItem === null) {
+        return state
+      }
```

Comment:
Do you mind making this section:
```javascript
  if (state.hoveredItem === null) {
     nextState = state
  } else {
    nextState = select(state, state.hoveredItem)
  }```
?
Having one return statement has been helpful during debugging.